### DISCUSSION TITLE
Our new name - a poll

### DISCUSSION THREAD

**Discussion by peterschutt** [litestar-org/litestar#800] [Polls]
# NOTE: We took a poll in discord among contributors and maintainers - the vote was not to change names (15 to 1 at time of update).

Adding options to a GitHub poll destroys existing votes. As we are still gathering nominations from members of the organization, it is likely that if you vote now, it will be lost.

This notice will be changed once we have all nominations, or the deadline for nominations is passed.

---

Starlite is to be renamed, see #799 for details.

The option with the most votes polled as at 2022-11-23 23:59:59 UTC will be our new name!

All members of the Starlite-API org may suggest up to two names for the poll, and entries close 2022-11-20 23:59:59 UTC.

Voting is open to everyone!

**Comment by peterschutt** [comment]
Hi @cofin @seladb @jtraub @dkress59 @infohash @jonadaly - feel free to nominate one or two names to add to the shortlist before we open the polls. Comment below, or in discord and I'll add to the list.

Cheers!

**Reply by Goldziher** [reply]
Checkout the discussion we have here as well: https://github.com/starlite-api/starlite/discussions/799

**Comment by dkress59** [comment]
Hey @peterschutt , I'd like to submit "blast" to the list – because it's BLazing fAST. Another idea is "starburst". Looking forward to doing the rebranding!

**Reply by peterschutt** [reply]
Hey mate, check this out: https://discord.com/channels/919193495116337154/919193688775749662/1043697395021656094

**Reply by dkress59** [reply]
lol ok, nevermind. I'm in hospital soon obviously missed the discussion 😆

**Comment by infohash** [comment]
Starlite, Stylight.

Skyline, Skylight, Skylite, Stylite.
